index.html

```diff
@@ -1,182 +1,258 @@
+
+      <ul class="lang">
         <li class="css">css</li>
         <li class="liner">|</li>
         <li class="html">html</li>
         <li class="liner-2">|</li>
         <li class="boot">bootstrap</li>
         <li class="liner-3">|</li>
         <li class="ruby">Ruby</li>
-    </ul>
-    
-    <button class="see" type="button">See Project</button>
-    
-    <div class="project">
-        <img class="sample" src="./images/sample-image.png" alt="Project Sample Image">
+      </ul>
+
+      <button class="see" type="button">See Project</button>
+
+      <div class="project">
+        <img
+          class="sample"
+          src="./images/sample-image.png"
+          alt="Project Sample Image"
+        />
         <h3 class="title">Multi-Post Stories</h3>
-        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
-            dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+        <p class="text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
         </p>
-    </div>
-    
-    <ul class="lang">
+      </div>
+
+      <ul class="lang">
         <li class="css">css</li>
         <li class="liner">|</li>
         <li class="html">html</li>
         <li class="liner-2">|</li>
         <li class="boot">bootstrap</li>
         <li class="liner-3">|</li>
         <li class="ruby">Ruby</li>
-    </ul>
-    
-    <button class="see" type="button">See Project</button>
+      </ul>
 
- </section>
+      <button class="see" type="button">See Project</button>
+    </section>
 
-   
-</body>
-</html>
\ No newline at end of file
+    <section>
+      <div class="contact-form-container">
+        <h2 class="contacts-title">CONTACT US</h2>
+        <p class="contact-texts">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      </div>
+
+      <div class="forms">
+        <form action="https://formspree.io/f/xdobrapq" method="post">
+          <ul>
+            <li>
+              <input
+                type="text"
+                placeholder="Text Input"
+                required
+                maxlength="30"
+              />
+            </li>
+            <li>
+              <input
+              class="text-email"
+                type="email"
+                placeholder="jeremy@gmail.com"
+                required
+                maxlength="30"
+              />
+            </li>
+            <li>
+              <textarea
+                name="message"
+                id="msg"
+                maxlength="500"
+                placeholder="Enter text here"
+              ></textarea>
```

Comment:
- [ ] Kindly make sure your `textarea` tag has a `required` attribute as stated in the project requirement. Because of the lack of  the ` required`  attribute, we get an error when submitting the form 🙏 

![image](https://user-images.githubusercontent.com/74814780/177401366-b29f661c-3d79-4725-84be-f72fc04d16c0.png)

Reply:
Hello @Ugboaja-Uchechi/ This issue is actually working on my side and tested it with my pair programming team. I added the required text but it still worked without it. Thank you for the review though
![review](https://user-images.githubusercontent.com/83514256/177405719-c95eb7a0-070a-49cb-b109-02c323be744e.PNG)
.